graphviz.spec

```diff
@@ -213,6 +226,22 @@ This package provides the PHP extension for %{name}.
 
 #-------------------------------------------------------------------------
 
+%if %{with python2}
+%package -n python2-graphviz
+Summary:        Graphviz bindings for python
+Group:          System/Libraries
+BuildRequires: python2-devel
+
+%description -n python2-graphviz
+This package provides the Python2 extension for %{name}.
+
+%files -n python2-graphviz
+%{_libdir}/graphviz/python2/*
+%{_libdir}/python2.7/site-packages/*
```

Comment:
Let's not hardcode 2.7 here -- if there's ever a 2.8, this will break the build.
%{_libdir}/python2*/site-packages/* or even %{py2_platsitedir}/* will keep working

Reply:
Yep changed